css/css-fonts/font-palette-36.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>font-palette multiple family-name and @font-palette-values</title>
+<link rel="help" href="https://drafts.csswg.org/css-fonts/#font-palette-prop">
+<link rel="help" href="https://drafts.csswg.org/css-fonts-4/#font-palette-values">
+<link rel="match" href="font-palette-36-ref.html">
```

Comment:
Suggest to add an 
`<meta name="assert"...` line, compare 
https://web-platform-tests.org/writing-tests/reftest-tutorial.html#id2

Reply:
Added